engine/src/exec-strings.cpp

```diff
@@ -1919,7 +1919,7 @@ void MCStringsEvalByteOffset(MCExecContext& ctxt, MCDataRef p_chunk, MCDataRef p
     // SN-2014-09-05: [[ Bug 13346 ]] byteOffset is 0 if the byte is not found, and 'empty'
     // is by definition not found; getting in the loop ensures at least 1 is returned.
     
-    if (!MCDataIsEmpty(p_chunk) && MCDataFirstIndexOf(p_string, p_chunk, MCRangeMake(p_start_offset, UINDEX_MAX), t_result))
+    if (!MCDataIsEmpty(p_chunk) && !MCDataIsEmpty(p_string) && MCDataFirstIndexOf(p_string, p_chunk, MCRangeMake(p_start_offset, UINDEX_MAX), t_result))
```

Comment:
@runrevmark I believe that's correct; no byte can be found in the empty bytestring. We should add a test to the LCB test suite to cover that.